### PR TITLE
Create journal-of-hip-preservation-surgery.csl

### LINKED ISSUE #1717: Journal of Hip Preserving Surgery
https://forums.zotero.org/discussion/37265/style-request-journal-of-hip-preserving-surgery/#Item_4

Guidelines for authors: http://www.oxfordjournals.org/our_journals/jhps/for_authors/manuscript_instructions.html

Online ISSN 2054-8397 

I just realized that it should be a comparable style like Clinical Orthopeadics and Related Research (CORR).

With the following changes:

1. truncated page numbers, i.e. type 345-9
2. For references with >4 authors, the first 3 authors should be listed, followed by et al.
3. Numbered in order of appearance
4. The journal should be written like that: Arch Orthop Trauma Surg. and not like that: Arch. Orthop. Trauma Surg.

### PR DESCRIPTION
**Pull request by zuphilip**
This fixes #1717

### CODE REVIEW

File: dependent/journal-of-hip-preserving-surgery.csl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>Journal of Hip Preserving Surgery</title>
```
Comment (rmzelle) [inline dependent/journal-of-hip-preserving-surgery.csl]: @zuphilip, the title seems to be "Journal of Hip Preservation Surgery" (not "preserving")

File: dependent/journal-of-hip-preserving-surgery.csl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <id>http://www.zotero.org/styles/journal-of-hip-preserving-surgery</id>
+    <link href="http://www.zotero.org/styles/journal-of-hip-preserving-surgery" rel="self"/>
+    <link href="http://www.zotero.org/styles/oxford-university-press-scimed-numeric" rel="independent-parent"/>
+    <link href="http://www.oxfordjournals.org/our_journals/jhps/for_authors/manuscript_instructions.html" rel="documentation"/>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>2054-8397</issn>
```
Comment (rmzelle) [inline dependent/journal-of-hip-preserving-surgery.csl]: (this is the electronic ISSN; see http://jhps.oxfordjournals.org/ )

File: dependent/journal-of-hip-preserving-surgery.csl (hunk 1 above)
Comment (zuphilip) [inline dependent/journal-of-hip-preserving-surgery.csl]: Good catch! Thank you! I just copied it from the forum resp. issue.

**Comment by zuphilip** [comment]
The PR is updated. Let me know if you have other concerns here.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**